src/components/Inventory/view/BatchDetailsModal.tsx

```diff
@@ -141,16 +167,71 @@ export default function BatchDetailsModal(props: BatchDetailsModalProps): JSX.El
           ]}
           scrolled={true}
         >
-          <Grid container item xs={12} spacing={2} textAlign='left'>
-            <Grid xs={12} padding={theme.spacing(1, 0, 1, 2)}>
-              <NurseryDropdown
-                label={strings.NURSERY_REQUIRED}
-                record={record}
-                setRecord={setRecord as unknown as React.Dispatch<React.SetStateAction<Batch>>}
-                organization={organization}
-                validate={validateFields}
-              />
+          {record.id !== -1 && (
+            <Grid container item xs={12} spacing={2} textAlign='left'>
+              <Grid item xs={gridSize()} sx={marginTop} paddingRight={paddingSeparator}>
+                <Textfield
+                  id='scientificName'
+                  value={speciesSelected?.scientificName}
+                  onChange={onChange}
+                  type='text'
+                  label={strings.SPECIES}
+                  display={true}
+                />
+              </Grid>
+              <Grid item xs={gridSize()} sx={marginTop} paddingLeft={paddingSeparator}>
+                <Textfield
+                  id='commonName'
+                  value={speciesSelected?.commonName}
+                  onChange={onChange}
+                  type='text'
+                  label={strings.COMMON_NAME}
+                  display={true}
+                />
+              </Grid>
+              <Grid item xs={gridSize()} sx={marginTop} paddingRight={paddingSeparator}>
+                <Textfield
+                  id='seedilingBatch'
+                  value={record.batchNumber}
+                  onChange={onChange}
+                  type='text'
+                  label={strings.SEEDLING_BATCH}
+                  display={true}
+                />
+              </Grid>
+              <Grid item xs={gridSize()} sx={marginTop} paddingLeft={paddingSeparator}>
+                <Typography sx={{ color: '#5C6B6C', fontSize: '14px' }}>{strings.ACCESSION_ID}</Typography>
```

Comment:
can color be `theme.palette.TwClrTxtSecondary`?

Reply:
I got that from [this spreadsheet](https://docs.google.com/spreadsheets/d/1nAaoqbf8DoSoLI389HYXHcZya3NnzG1RFUeYSBr3GQA/edit#gid=324615937). It's the only #5C6B6C on there

Reply:
Oh, sure!